mobile/src/main/java/org/hawkular/client/android/backend/BackendPipes.java

```diff
@@ -47,6 +47,7 @@ private Names() {
         public static final String OPERATION_PROPERTIES = "operationProperties";
         public static final String PERSONA = "persona";
         public static final String TRIGGERS = "triggers";
+        public static final String UPDATE_TRIGGER = "trigger-enabled";
```

Comment:
Hey. Please don't use `trigger-enabled` as String here as it is generic trigger update. Better use `trigger-update`